src/ViewRendererBundle/objects/helper/TPkgViewRendererLessCompiler.php

```diff
@@ -227,11 +243,8 @@ public function writeCssFileForPortal($css, $portal)
     {
         $lessDir = $this->getLocalPathToCompiledLess();
 
-        if (false === \is_dir($lessDir)) {
-            if (!mkdir($lessDir, 0777, true) && !\is_dir($lessDir)) {
-                return false;
-            }
-        }
+        // NOTE this is probably already done in getGeneratedCssForPortal() for the cache of Less itself
+        $this->checkAndCreateDir($lessDir);
```

Comment:
If the method is changed to throw an exception, we should catch it here and return false (don't change the method signature in a fix release).

Reply:
ok